### ISSUE TITLE
Memory leak on call to locals() when script is obfuscated in super mode

### ISSUE THREAD

**Issue by 2rgman** [dashingsoft/pyarmor#550]
Hi

**Bug description:**
Inside our script we have a loop that call a function at fixed interval
Inside the function we test if locals() contains a variable
If we obfuscate our script in super mode, a memory leak occur when running the script

After some research we have managed to find the cause and created a short code example which reproduce the problem

**How to reproduce:**
Obfuscate this script in super mode, run it and monitor the memory usage:
```
from time import sleep
def leak():
    array = bytearray(1024*1024)
    if 'some_var' in locals():
        pass
while True:
    leak()
    sleep(1)
```
On our side this code make python.exe memory usage to grow in 1MB every second
And this happen only if the script is obfuscated in super mode (On `--advanced 1` there is no leak)

**More details:**
PyArmor Version 6.7.3 - Purchased
Python 3.7.8
Build machine is Windows 7 32bit but verified on Windows 10 64bit too
Obfuscation command line: `pyarmor obfuscate --output dist/obftest --advanced 2 SuperModeLeakTest/test.py`
Tested on build machine itself with same python version (but reproducible on other machines too)

If you need more info please tell me

Thanks!

**Comment by jondy** [comment]
I'll check it

**Comment by jondy** [comment]
Your test script is very useful to find the problem, I have fixed it. Please verify it in the pre-relase version v6.7.4
https://pyarmor.readthedocs.io/en/latest/change-logs.html#dev

**Comment by jondy** [comment]
Fix it in v6.7.4
